src/App.jsx

```diff
@@ -1,34 +1,26 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import react from 'react'
+import { BrowserRouter , Route,Routes} from "react-router-dom";
+import ArticlesPage from './Components/ArticlesPage'
+
+import React from 'react';
 import './App.css'
+import Header from './Components/Header';
+import Footer from './Components/Footer';
+
 
-function App() {
-  const [count, setCount] = useState(0)
 
+function App() {
+  
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <BrowserRouter>
+      <Header />
+      <Routes>
+        <Route path="/articles" element={<ArticlesPage/>} />
```

Comment:
I knew this was the path after looking at the code, but a regular user wouldn't! Please make the articles to render on path `/` or have a home page which links to `/articles`